src/test/resources/Room.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Confirming the functionality of the room top column
+
+  Scenario Outline: Validating the functionality of 'Room' column
+    Given The user has logged in the website
+    When The user clicks on the "Rooms" top column
+    And The user clicks on the Click to Select Dates tab
+    Then User validates calendar is present
+
+    When User selects the start date and end date
+
+    And The user clicks the 'Apply' button
+    Then User validates the dates on input tab are start date and end date
+
+    When User clicks footer icon with value 'phone'
```

Comment:
this must be a different test with its own feature file. the footer is on all pages its not a room's plage feature

Reply:
Noted